bftengine/tests/bcstatetransfer/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(bcstatetransfer_tests
```

Comment:
If you also add `add_test(bcstatetransfer_tests bcstatetransfer_tests)`, then this will get run by a `make test`.

Reply:
thanks!